src/Renderer/Image.php

```diff
@@ -73,18 +72,14 @@ function render(Frame $frame)
             $parts = preg_split("/\s*\n\s*/", $msg);
             $font = $style->font_family;
             $height = 10;
-            $_y = $alt ? $y + $h - count($parts) * $height : $y;
+            $offset = $alt !== "" ? $h : 0;
 
-            foreach ($parts as $i => $_part) {
-                $this->_canvas->text($x, $_y + $i * $height, $_part, $font, $height * 0.8, [0.5, 0.5, 0.5]);
+            foreach ($parts as $i => $part) {
+                $this->_canvas->text($x, $y + $offset + $i * $height, $part, $font, $height * 0.8, [0.5, 0.5, 0.5]);
```

Comment:
I wonder if it's worth maintaining this bit of logic. Dompdf used to return specific messaging when an image couldn't be loaded, but all we do now is [return a generic message](https://github.com/dompdf/dompdf/blob/master/src/Image/Cache.php#L173).

Reply:
True, displaying the message doesn’t seem that useful anymore. I pushed an extra commit just removing the logic. What do you think?

Reply:
Sample document:

<details>
<summary>HTML</summary>

```html
<!DOCTYPE html>
<html>

<head>
<meta charset="UTF-8">
<style>
@page {
    size: 400pt 300pt;
    margin: 50pt;
}

p {
    margin: 0 0 10pt;
}

.i1 {
    width: 100%;
    height: 0;
    outline: 2pt solid green;
}

.i2 {
    width: 50%;
    height: 100pt;
    outline: 2pt solid green;
}

.i3 {
    width: 100%;
    height: 0;
    outline: 2pt solid green;
}
</style>
</head>

<body>
    <p><img src="invalid.png" class="i1" alt=""></p>
    <p><img src="invalid.png" class="i2" alt=""></p>
    <p><img src="invalid.png" class="i3" alt="Alternate text"></p>
    <p><img src="invalid.png" class="i3" alt="0"></p>
    <p><img src="invalid.png" class="i3" alt="Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua."></p>
</body>

</html>
```
</details>

Before: [before.pdf](https://github.com/dompdf/dompdf/files/13190774/before.pdf)
After: [after.pdf](https://github.com/dompdf/dompdf/files/13190773/after.pdf)
2.0.3 for comparison: [203.pdf](https://github.com/dompdf/dompdf/files/13190771/203.pdf)

Reply:
Yeah, I think excluding it makes sense. It's a much cleaner look, particularly if alt text is provided.